ethpm/package.py

```diff
@@ -73,3 +79,20 @@ def name(self):
     @property
     def version(self):
         return self.package_data['version']
+
+    #
+    # Deployments
+    #
+
+    def get_deployments(self, w3):
```

Comment:
Do we always require a `w3` instance here to get passed in? or should it be able to use the default if there is one?

Reply:
I think that it's ok to make it optional `get_deployments(self, w3=None)` and to default to the class `w3` instance if no instance is passed in.